docs/Makefile

```diff
@@ -1,4 +1,42 @@
-# Minimal makefile for Sphinx documentation
+all:  pbench-agent.html test-rpms.html
+
+everything: all pbench-agent.pdf help
+
+%.html:	%.org
+	emacs -batch -l batch-export.el --eval "(batch-org-export-as 'html)" $<
+
+%.pdf:	%.tex
+	latexmk --shell-escape -pdf -xelatex $<
+
+%.tex: %.org
+	emacs -batch -l batch-export.el --eval "(batch-org-export-as 'latex)" $<
+
+desthost = $(shell getconf.py repohost $(prog) repo)
+destuser = $(shell getconf.py repouser $(prog) repo)
+destdir  = /var/www/html
+dest = ${destuser}@${desthost}:${destdir}
+
+push-doc:	pbench-agent.html
+	rsync -ave ssh $< ${dest}/index.html
+
+push-results:	results.html
+	rsync -ave ssh $< ${dest}/$<
+
+push-test-rpms:	test-rpms.html
+	rsync -ave ssh $< ${dest}/test-repo/$<
+
+push-relnotes: RELEASE-NOTES.html
+	rsync -ave ssh $< ${dest}/doc/$<
+
+.PHONY: clean veryclean
+clean:
+	rm -f *~ *.fdb_latexmk *.fls *.toc *.aux *.log *.out
+
+veryclean:	clean
+	rm -f *.html *.pdf
```

Comment:
I don't see `*.tex` in either of these lists...should it be there?

Reply:
I'll add it, but I'll also defer to @ndokos for the final word on this.

Reply:
And `*.xdv` should be there as well.

Reply:
`*.tex` should probably be in the `veryclean` action. For debugging purposes, it is useful to do a `clean` and then process the `tex` file again to see where it breaks (e.g. the missing `.sty` files).

OTOH, we've never made PDFs available, only HTML, so one solution is to get rid of the `tex/pdf` generation path altogether.

Reply:
Not to tattle or anything, but `*.tex` is currently in the `clean` target, not the `veryclean` target....